numpy/_core/fromnumeric.py

```diff
@@ -1306,6 +1306,8 @@ def argmax(a, axis=None, out=None, *, keepdims=np._NoValue):
 
     Indexes of the maximal elements of a N-dimensional array:
 
+    >>> a.flat[np.argmax(a)]
```

Comment:
This seems like a useful addition to the docs, but the example code does not match the text above (e.g. `Indexes of the maximal elements of a N-dimensional array:`). Maybe move this a bit down (after the `a[ind]` example)? Or rename the text to `Indices and values of the maximal ...`

Note: both Indexes and Indices are correct, but it is a bit inconsistent to have the docs of `argmin` and `argmax` use a different form.

Reply:
That seems to me like it could be a separate change. The addition just shows how to simply use the index without having to unravel it first; the raw form is used to actually to get the value, but that was already the case with the unraveled form. (and unlike the unraveled form, the raw index already appears above, so seemed to me like it's not worth repeating).